### ISSUE TITLE
Remove the O Series

### ISSUE THREAD

**Issue by fabiand** [kubevirt/common-instancetypes#262]
**What happened**:
There is low value n the O Series. Let's remove it.
KubeVirt gained memory overcommit, an admin sided feature, that's easier and more consistent to use.

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:
Steps to reproduce the behavior.

**Additional context**:

**Environment**:
- KubeVirt version (use `virtctl version`): N/A
- Kubernetes version (use `kubectl version`): N/A
- VM or VMI specifications: N/A
- Cloud provider or hardware configuration: N/A
- OS (e.g. from /etc/os-release): N/A
- Kernel (e.g. `uname -a`): N/A
- Install tools: N/A
- Others: N/A

**Comment by lyarwood** [comment]
@fabiand I believe that support pre-dates the work @vladikr did in https://github.com/kubevirt/kubevirt/pull/9799 that is used by this instance type class. I'm not against removing it I just wanted to check with you both that we still want to deprecate this as I'll also remove support from the `instancetype.kubevirt.io` API.

**Comment by fabiand** [comment]
I would remove the O Series.

But I do not understand `remove support from the instancetype.kubevirt.io API.` could you rephrase this for me?

Also: We might want to add the "graduation level" to instanceTypes

**Comment by lyarwood** [comment]
> I would remove the O Series.
> 
> But I do not understand `remove support from the instancetype.kubevirt.io API.` could you rephrase this for me?

https://github.com/kubevirt/kubevirt/pull/9799 specifically introduced the `spec.memory.overcommitPercent` attribute used by this class. If it's no longer used here I was going to deprecate it in `v1beta1` to potentially drop ahead of `v1` in the future.

> Also: We might want to add the "graduation level" to instanceTypes

Possibly, we could definitely use it for preferences for non-GA'd OS's etc.
